src/lib.rs

```diff
@@ -164,7 +186,8 @@ impl Contract {
         editor == env::current_account_id() || editor == post.author_id
     }
 
-    pub fn edit_post(&mut self, id: PostId, body: PostBody, labels: HashSet<String>) {
+    #[payable]
+    pub fn edit_post(&mut self, id: PostId, body: PostBody, labels: HashSet<String>) -> Promise {
```

Comment:
Same thing, I don't think we need to wait on the other contract to execute, so we can probably drop the promise without returning it.

Reply:
Sounds good!